### PR TITLE
quick and dirty pivot

### PR DESCRIPTION
**Pull request by Awpteamoose**
## Description

Add a pivot field to UiTransform.

## Modifications

`UiTransform::new` now accepts another `Anchor` as an argument

## PR Checklist

By placing an x in the boxes I certify that I have:

- [x] Ran `cargo test --all` locally if this modified any rs files.
- [x] Ran `cargo +stable fmt --all` locally if this modified any rs files.
- [x] Updated the content of the book if this PR would make the book outdated.
- [x] Added a changelog entry if this will impact users, or modified more than 5 lines of Rust that wasn't a doc comment.
- [x] Added unit tests for new APIs if any were added in this PR.
- [x] Acknowledged that by making this pull request I release this code under an MIT/Apache 2.0 dual licensing scheme.

### CODE REVIEW

**Comment by azriel91** [comment]
that visual demonstration, so good 👌

**Comment by Awpteamoose** [comment]
@azriel91 done, also fixed book

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ extra bounds from `AnimatablePrefab` and `AnimationSetPrefab` ([#1435])
 * Added generic parameter type to `Transform` to configure floating point precision. ([#1334])
 * `NetConnection` is automatically created when client starts sends data to server. ([#1539])
 * User will receive `NetEvent::Connected` on new connection and `NetEvent::Dissconnected` on disconnect. ([#1539])
+* Added a `pivot` field to `UiTransform`. ([#1571])
 
 ### Removed
 - Removed all `NetEvent's` because they were not used. ([#1539])
```
Comment (jojolepro) [inline docs/CHANGELOG.md]: You forgot to add the link too :(

Comment (Awpteamoose) [reply docs/CHANGELOG.md]: woops fixed

**Pull request merged**